Swiss statistics days 2024 (#139)
SwissDataScienceCenter/renku-blog: blog/2024-09-11-swiss-statistics-days/index.mdx

---
slug: sst-2024
title: |
  Swiss Statistics Meeting - Renku: a platform for transparent and reproducible data-centric work
authors: [elisabet, rokroskar, laura, jimena]
tags:
  - Swiss Statistics Meeting
  - Poster

> image: ./image_1.png

Reviewer: Image for SEO/link previews should be landscape.

description: |
  Renku was presented in the Swiss Statistics Meeting, which took place on Sept. 4th

>   and 5th 2024 in Aarau (Switzerland). This meeting gathered specialists related to 

Reviewer 2: ```suggestion
  and 5th 2024 in Aarau (Switzerland) and brought together specialists related to 
```

>   statiscis domain across private and public sectors. The SDSC contribution “Renku: a platform 

Reviewer 2: ```suggestion
  statistics domain across private and public sectors. The SDSC contribution “Renku: a platform 
```

  for transparent and reproducible data-centric work” focused on those features in 
  Renku 2.0 that make it a unique platform to support transparent data-centric work, 
  since it connects data sources and code repositories, and enables launching sessions, 
  where data scientists, statisticians and other practitioners can work in their prefererred
  compute environment (Python, R, Matlab, etc.) and share their work.
---

![Renku in Swiss Statistics Meeting 2024](./image_1.png)

> <!-- truncate -->

Reviewer: Honestly, you should truncate after the first paragraph. This does not look engaging at all:

![Screenshot 2024-09-23 at 09 18 44](https://github.com/user-attachments/assets/aaac6811-07a4-42c9-b5d3-f65cfcbe0401)

The [Swiss Statistics Meeting](https://stat.ch/en/swiss_statistics_meeting) took place

> on Sept. 4th and 5th 2024 in Aarau (Switzerland). It gathered specialists related to

Reviewer 2: ```suggestion
on Sept. 4th and 5th 2024 in Aarau (Switzerland) and brought together specialists related to
```

the statistics domain across private and public sectors. Multiple presentations contributed
to discuss developments in research and methodological questions related to data across
the data lifecycle, including topics such as data sources,
data collection and management, data analysis, open data, and data security among others.

Our contribution [“Renku: a platform for transparent and reproducible data-centric work”](https://drive.google.com/file/d/1qYsrPm4rm3Osjmjvmoz0RIgWyxWN335F/view?usp=share_link)
focused on those Renku features in Renku 2.0 that make it a unique platform to support
transparent data-centric work. Renku connects to public and private data sources, and to

> code repositories. Thus, Renku 2.0 can launch different types of sessions on top of them,

Reviewer 2: ```suggestion
code repositories. These data and code come to life in Renku sessions,
```

where data scientists, statisticians and other practitioners can work in their prefererred
compute environment (Python, RStudio, Matlab, VSCode or others ) with the compute infrastructure

> that they select. Having these three components under the same umbrella makes it easier to

Reviewer 2: ```suggestion
that they select. Having data, code, and compute under the same umbrella makes it easier to
```
